src/dataset.hpp

```diff
@@ -62,7 +62,13 @@ struct randomx_cache {
 //A pointer to a standard-layout struct object points to its initial member
 static_assert(std::is_standard_layout<randomx_dataset>(), "randomx_dataset must be a standard-layout struct");
 //the following assert fails when compiling Debug in Visual Studio (JIT mode will crash in Debug)
+#if _MSC_VER && !__INTEL_COMPILER && _DEBUG
+#define _STR(x) #x
```

Comment:
Identifiers starting with an underscore are reserved for the compiler. Please rename.

Reply:
Done